cmd/applyconfig/applyconfig.go

```diff
@@ -302,7 +303,7 @@ func init() {
 
 func main() {
 	o := gatherOptions()
-	var adminErr, standardErr error
+	var adminErrs, standardErrs []error
```

Comment:
These don't seem to be used directly, they could be replaced by a single `bool`.

Reply:
Done